src/functions/carts/editProductQuantityInCart/handler.ts

```diff
@@ -0,0 +1,24 @@
+import 'source-map-support/register';
+
+import type { ValidatedEventAPIGatewayProxyEvent } from '@libs/apiGateway';
+import { middyfy } from '@libs/lambda';
+import Response from '@lamdaModel/lambdaResponse';
+import CartRow from '@dbModel/tables/cart';
+import editProductQuantityInCart from '@carts/editProductQuantityInCart/function';
+import HttpStatusCodes from '@lamdaModel/httpStatusCodes';
+import schema from '@schema/lambdaSchema/editProductQuantityInCart';
+
+const editProductQuantityInCartHandler: ValidatedEventAPIGatewayProxyEvent<typeof schema> = async (event) => {
```

Comment:
I think it's better to rename this function in `editCartHandler`, because now only quantity could be changed, but in the future we'll probably add some other editable field.
For example, a product version.